core/src/main/scala/Github.scala

```diff
@@ -76,13 +76,13 @@ object Github {
   ) extends Event
 
   final case class ReleaseEvent(
-    id: Long,
+    id: String,
```

Comment:
One trick we can try is parameterize it into `ReleaseEvent[A]` so we have `ReleaseEvent[Long]` and `ReleaseEvent[String]`.. wdyt? @timperrett @kaiserpelagic